.github/workflows/spread.yaml

```
@@ -31,7 +31,9 @@ jobs:
       fail-fast: false
       matrix:
         spread:
```

Would a reference in the other direction also be useful?
```suggestion
        # from the backends defined in the top-level `spread.yaml`
        spread:
```

---

Good idea!